_includes/pbc/all/install-features/202505.0/ssp/install-ssp-service-management-features.md

```diff
@@ -0,0 +1,1183 @@
+use Spryker\Zed\ProductStorage\ProductStorageDependencyProvider as SprykerProductStorageDependencyProvider;
+
+class ProductStorageDependencyProvider extends SprykerProductStorageDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ProductStorageExtension\Dependency\Plugin\ProductConcreteStorageCollectionExpanderPluginInterface>
+     */
+    protected function getProductConcreteStorageCollectionExpanderPlugins(): array
+    {
+        return [
+            new ShipmentTypeProductConcreteStorageCollectionExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Quote/QuoteDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Quote;
+
+use SprykerFeature\Zed\SspServiceManagement\Communication\Plugin\Quote\ServicePointQuoteExpanderPlugin;
+use SprykerFeature\Zed\SspServiceManagement\Communication\Plugin\Quote\SspShipmentTypeQuoteExpanderPlugin;
+use Spryker\Zed\ShipmentTypeCart\Communication\Plugin\Quote\ShipmentTypeQuoteExpanderPlugin;
+
+class QuoteDependencyProvider extends SprykerQuoteDependencyProvider
+{
+
+    /**
+     * @return list<\Spryker\Zed\QuoteExtension\Dependency\Plugin\QuoteExpanderPluginInterface>
+     */
+    protected function getQuoteExpanderPlugins(): array
+    {
+        return [
+            new SspShipmentTypeQuoteExpanderPlugin(),
+            new ServicePointQuoteExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Sales/SalesDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Sales;
+
+use Spryker\Zed\Sales\SalesDependencyProvider as SprykerSalesDependencyProvider;
+use SprykerFeature\Zed\SspServiceManagement\Communication\Plugin\Sales\ProductTypeOrderItemsPostSavePlugin;
+use SprykerFeature\Zed\SspServiceManagement\Communication\Plugin\Sales\ScheduleTimeOrderItemExpanderPreSavePlugin;
+
+class SalesDependencyProvider extends SprykerSalesDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\SalesExtension\Dependency\Plugin\OrderItemExpanderPreSavePluginInterface>
+     */
+    protected function getOrderItemExpanderPreSavePlugins(): array
+    {
+        return [
+            new ScheduleTimeOrderItemExpanderPreSavePlugin(),
+        ];
+    }
+    
+     /**
+     * @return array<\Spryker\Zed\SalesExtension\Dependency\Plugin\OrderItemsPostSavePluginInterface>
+     */
+    protected function getOrderItemsPostSavePlugins(): array
+    {
+        return [
+            new ProductTypeOrderItemsPostSavePlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. In the Back Office, go to **Catalog** > **Products**.
+2. Create an abstract product with a **service** product type.
+3. For the abstract product you've created, create a concrete product with the following settings:
+  * Enable **Service Date and Time**
+  * Add **Delivery** and **On-Site Service** shipment types
+  
+
+### B. Set Up Product Offers for the Service Product
```

Comment:
Provide more specific instructions, not sure how to do it.

Reply:
This is the high level list )
The instructiction are below)